server/services/documentService.test.ts

```diff
@@ -0,0 +1,21 @@
+import { RPClient } from '../data'
+import DocumentService from './documentService'
+import * as UserContext from '../middleware/userContextMiddleware'
+
+describe('Document Service', () => {
```

Comment:
(Unrelated to code change - just added to increase code coverage and make the pipeline happy)